sdks/python/apache_beam/io/avroio.py

```diff
@@ -511,7 +488,9 @@ def __init__(
         end in a common extension, if given by file_name_suffix. In most cases,
         only this argument is specified and num_shards, shard_name_template, and
         file_name_suffix use default values.
-      schema: The schema to use, as returned by avro.schema.Parse
+      schema: The schema to use (dict).
```

Comment:
Does fastavro support a more structured definition for schemas or does is have to be a dict ?

Reply:
According to https://fastavro.readthedocs.io/en/latest/schema.html, it's a dict.